spyder/plugins/editor/plugin.py

```diff
@@ -2184,30 +2184,14 @@ def load(self, filenames=None, goto=None, word='',
                 selectedfilter = ''
 
             if not running_under_pytest():
-                # See: spyder-ide/spyder#3291
```

Comment:
Did you check this is no longer an issue, right?

Reply:
Actually, this should not be removed. Thanks for catching that.